schemas/ispyb/updates/2021_04_01_ContainerType_insert.sql

```diff
@@ -0,0 +1,13 @@
+INSERT IGNORE INTO SchemaStatus (scriptName, schemaStatus) VALUES ('2021_04_01_ContainerType_insert.sql', 'ONGOING');
+
+INSERT INTO ContainerType (containerTypeId, name, capacity, proposalType, active) 
+  VALUES 
+    (27, 'Basket', NULL, 'mx', 0),   -- capacity varies, used by i24 most recently in 2016
+    (28, 'Cane', NULL, 'mx', 0),     -- capacity varies, assumed mx, used most recently in 2013
+    (29, 'Terasaki72', 72, 'mx', 0), -- used by i24 most recently in 2017
+    (30, 'Puck-16', 16, 'mx', 1),    -- used by i24 most recently in 2020
+    (31, 'Block-4', 16, 'mx', 1),    -- used by i23 most recently in 2021, all have capacity 16
+    (32, 'Box', 25, 'xpdf', 1), 
+    (33, 'Puck', 22, 'xpdf', 1);
```

Comment:
For xpdf is it worth calling their puck Puck-22. What if they end up using a 16 pin puck later?
Is Cane really worth adding if its that old and not consistent?

Reply:
Oops, I have somehow accidentally merged this into master!

To answer your questions:

- Good idea re: Puck-22.
- Cane is old, yes, but when we drop the `Container.containerType` column at some point, then I don't want to lose historical data. The idea would be to update the historical rows so that they reference the ContainerType table instead before dropping the column.

Reply:
I've now renamed the XPDF Puck to Puck-22.